packages/code-utils/src/opcodes.ts

```diff
@@ -75,6 +75,7 @@ const codes = {
   0x5b: "JUMPDEST",
 
   // 0x60 & 0x70 range - pushes
+  0x5f: "PUSH0",
```

Comment:
The comment above is confusing.

Reply:
Yeah, didn't really know what to change it to though... (thoughts?)

Reply:
`Beginning of push op codes` maybe?